### ISSUE TITLE
missing files on install 5.2 on max Big Sur

### ISSUE THREAD

**Issue by pdm76** [spyder-ide/spyder#16959]
AppleIntelHD5000GraphicsGLDriver: #2: No such file or directory
Dec  4 12:25:34 Pauls-MacBook-Pro Spyder[1741]: getattrlist failed for /System/Library/Frameworks/OpenGL.framework/Resources//GLRendererFloat.bundle/GLRendererFloat: #2: No such file or directory
Dec  4 13:29:47 Pauls-MacBook-Pro com.apple.xpc.launchd[1]: Coalition Cache Hit: app<application.org.spyder-ide.8698188957.8698189397(501)> [1303]
Dec  4 13:29:51 Pauls-MacBook-Pro Spyder[3237]: getattrlist failed for /System/Library/Extensions/AppleIntelHD5000GraphicsGLDriver.bundle/Contents/MacOS/AppleIntelHD5000GraphicsGLDriver: #2: No such file or directory
Dec  4 13:29:51 Pauls-MacBook-Pro Spyder[3237]: getattrlist failed for /System/Library/Frameworks/OpenGL.framework/Resources//GLRendererFloat.bundle/GLRendererFloat: #2: No such file or directory

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/Applications/Spyder 5.2.app/Contents/Resources/__boot__.py", line 106, in <module>
    _run()
  File "/Applications/Spyder 5.2.app/Contents/Resources/__boot__.py", line 84, in _run
    exec(compile(source, path, "exec"), globals(), globals())
  File "/Applications/Spyder 5.2.app/Contents/Resources/Spyder.py", line 3, in <module>
    start.main()
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/app/start.py", line 236, in main
    mainwindow.main(options, args)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/app/mainwindow.py", line 2036, in main
    mainwindow = create_window(MainWindow, app, splash, options, args)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/app/utils.py", line 308, in create_window
    CONF.notify_all_observers()
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/config/manager.py", line 301, in notify_all_observers
    self.notify_section_all_observers(section)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/config/manager.py", line 378, in notify_section_all_observers
    self.notify_observers(section, option)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/config/manager.py", line 347, in notify_observers
    self._notify_option(section, option, value)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/config/manager.py", line 358, in _notify_option
    observer.on_configuration_change(option, section, value)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/api/config/mixins.py", line 351, in on_configuration_change
    method(value)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/widgets/main_widget.py", line 745, in change_clients_autocall
    self._change_client_conf(
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/widgets/main_widget.py", line 892, in _change_client_conf
    client_conf_func(value)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/widgets/shell.py", line 407, in set_autocall
    self.execute(cmd.format(autocall), hidden=True)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/widgets/shell.py", line 257, in execute
    super(ShellWidget, self).execute(source, hidden, interactive)
  File "/Applications/Spyder 5.2.app/Contents/Resources/lib/python3.9/spyder/plugins/ipythonconsole/widgets/debugging.py", line 569, in execute
    return super(DebuggingWidget, self).execute(
  File "qtconsole/console_widget.pyc", line 647, in execute
  File "qtconsole/frontend_widget.pyc", line 285, in _execute
AttributeError: 'NoneType' object has no attribute 'execute'
```

## Versions

* Spyder version: 5.2.0 
* Python version: 3.9.5
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Darwin 20.6.0

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7   :  0.2.1 (OK)
atomicwrites >=1.2.0        :  1.4.0 (OK)
chardet >=2.0.0             :  4.0.0 (OK)
cloudpickle >=0.5.0         :  2.0.0 (OK)
cookiecutter >=1.6.0        :  1.7.3 (OK)
diff_match_patch >=20181111 :  20200713 (OK)
intervaltree >=3.0.2        :  3.1.0 (OK)
IPython >=7.6.0             :  7.29.0 (OK)
jedi >=0.17.2;<0.19.0       :  0.18.1 (OK)
jellyfish >=0.7             :  0.8.9 (OK)
jsonschema >=3.2.0          :  4.2.1 (OK)
keyring >=17.0.0            :  23.2.1 (OK)
nbconvert >=4.0             :  6.3.0 (OK)
numpydoc >=0.6.0            :  1.1.0 (OK)
parso >=0.7.0;<0.9.0        :  0.8.2 (OK)
pexpect >=4.4.0             :  4.8.0 (OK)
pickleshare >=0.4           :  0.7.5 (OK)
psutil >=5.3                :  5.8.0 (OK)
pygments >=2.0              :  2.10.0 (OK)
pylint >=2.5.0              :  2.11.1 (OK)
pylsp >=1.3.1;<1.4.0        :  1.3.1 (OK)
```

**Comment by ccordoba12** [comment]
@pdm76, thanks for reporting. Our bot closed incorrectly closed your issue.

To fix it, please install spyder-kernels **2.2.0** in the environment connect to where our app. This error will be fixed in our next version (5.2.1), to be release shortly.

**Comment by pdm76** [comment]
Installed spyder-kernels 2.2.0 and the problem was resolved. 
Thanks
